images/centos.yaml

```diff
@@ -378,6 +378,16 @@ actions:
   releases:
   - 9-Stream
 
+- trigger: post-unpack
+  action: |-
+    #!/bin/sh
+    # Remove security.selinux xattr.
+    find / -xdev -exec setfattr -x security.selinux {} 2>/dev/null \;
+  types:
+  - container
+  releases:
+  - 9-Stream
```

Comment:
I think that by now, the release specifier is potentially redundant. Something to cleanup another day maybe.